x/dsmr/storage_test.go

```diff
@@ -275,7 +278,7 @@ func TestRestartSavedChunks(t *testing.T) {
 	// 5. Pending local chunk
 	// 6. Pending remote chunk
 	numChunks := 6
-	storage, validChunks, _, restart := createTestStorage(t, numChunks, 0)
+	storage, validChunks, _, restart := createTestStorage(t, numChunks, 0, []int64{2, 2, 1, 1, 2, 2})
```

Comment:
It seems like this could easily break silently if we created a slice of different length than `numChunks` by accident.

Would it make sense to pass in a slice of expiry times for both valid/invalid?